src/firmware/riscv/blackhole/host_mem_address_map.h

```diff
@@ -5,6 +5,9 @@
 #include <cstdint>
 #include <stdint.h>
 
+// Remove inline ASAP
```

Comment:
"inline" is there so you don't have to change other tests? E.g. constants can be used as both blackhole::host_mem::address_map::CONST and host_mem::address_map::CONST ?

Reply:
Correct, don't have to immediately change tests or tt_metal.
I was thinking it would make it easier to remove this API if we first deprecate, because we have so much trouble moving UMD forward in metal.